sql/parse/parse.go

```diff
@@ -1172,6 +1174,31 @@ func convertUpdate(ctx *sql.Context, d *sqlparser.Update) (sql.Node, error) {
 	return plan.NewUpdate(node, updateExprs), nil
 }
 
+func convertLoad(ctx *sql.Context, d *sqlparser.Load) (sql.Node, error) {
+	unresolvedTable := tableNameToUnresolvedTable(d.Table)
+
+	var ignoreNumVal int8 = 0
+	var ok = false
+	if d.IgnoreNum != nil {
+		ignoreNum, err := convertVal(d.IgnoreNum)
+		if err != nil {
+			return nil, err
+		}
+
+		ign, err := ignoreNum.Eval(ctx, nil)
+		if err != nil {
+			return nil, err
+		}
+
+		ignoreNumVal, ok = ign.(int8)
+		if !ok {
+			return nil, fmt.Errorf("Cannot parse ignore Value")
+		}
+	}
+
+	return plan.NewLoadData(bool(d.Local), d.Infile, unresolvedTable, columnsToStrings(d.Columns), d.Fields, d.Lines, ignoreNumVal), nil
```

Comment:
This is a good first draft, but what you really need to do is create a new Insert node with the LoadData node as the data source. You need the Insert to be the top-level node so that other logic (like type conversions, trigger application, constraint checking) etc. just works. Otherwise you'll end up duplicating the insert logic anywhere, as a bunch of your TODOs mention)